src/lab4.py

```diff
@@ -0,0 +1,43 @@
+class Lamp:
+    def __init__(self, type_="Нема типу", power=0, quantity=0, name="None", price=0, character="Нема характер"):
+        self.__type = type_
+        self.__power = power
+        self.__quantity = quantity
+        self.__name = name
+        self.price = price
+        self.character = character
+
+    def get_type(self):
+        return self.__type
+
+    def get_power(self):
+        return self.__power
+
+    def get_quantity(self):
+        return self.__quantity
+
+    def get_name(self):
+        return self.__name
+
+    def __str__(self): 
+        return (f"Type: {self.__type} \nPower: {self.__power} W\n"
+                f"Quantity: {self.__quantity} \nName: {self.__name}\nPrice: {self.price}\nCharacter: {self.character}")
```

Comment:
Added a check for `None` in `__str__` to avoid printing "None" for the `name` attribute.